clientUtils/SqlFilterSExpression.ts

```diff
@@ -0,0 +1,515 @@
+import parse = require("s-expression")
+import pointer from "json8-pointer"
+import { isArray, tail, without } from "lodash"
+export type SExprAtom = string | String | SExprAtom[] // eslint-disable-line @typescript-eslint/ban-types
```

Comment:
hm, why do we need `String` (uppercase) in here?

Reply:
Good idea!